### PR TITLE
Update url dispatcher: url() to path()

### PR DESCRIPTION
**Pull request by bendikstoasannes**

### CODE REVIEW

File: chemie/events/urls.py [hunk 1]
```diff
@@ -64,54 +56,54 @@
...
         views.BedpresEnlistedUsersView.as_view(),
         name="adminlist_bedpres",
     ),
-    url(
-        r"^bedpres/adminlist/arrivalstatus/(?P<registration_id>[0-9]+)",
+    path(
+        "bedpres/adminliste/oppmotestatus/<int:registration_id>/",
         views.change_arrival_status,
         name="arrival_status_bedpres",
     ),
```
Comment (anjarfr) [inline chemie/events/urls.py]: Same here. must edit the hardcoded urls

File: chemie/customprofile/templates/customprofile/memberships.html [hunk 2]
```diff
@@ -99,7 +99,7 @@ <h5 class="heading">Klikk på et rødt ikon for å tildele 5 års medlemsskap fr
               var csrftoken = getCookie('csrftoken');
               $.ajax({
                   type: "GET",
-                  url: "medlemmer/endre/" + myID,
+                  url: "endre/" + myID,
```
[the remark this answers is not in the document]
Comment (bendikstoasannes) [reply chemie/customprofile/templates/customprofile/memberships.html]: The button works as intended when removing `medlemmer`. The default url is the current page (`profil/medlemmer`). So it is sufficient to only append `endre + myID`

File: chemie/customprofile/urls.py [hunk 3]
```diff
@@ -1,32 +1,28 @@
...
+    path("profil/registrer/", views.register_user, name="register"),
+    path("profil/rediger/", views.edit_profile, name="edit"),
+    path("profil/rediger-push-varsler/", views.edit_push, name="edit-push"),
+    path(
+        "profil/glemt-passord/", views.forgot_password, name="forgotpassword"
     ),
-    url(
-        r"^profile/aktiver/(?P<code>[a-z0-9]{32})/?",
+    re_path(
+        r"^profil/aktiver/(?P<code>[a-z0-9]{32})/?",
```
[the remark this answers is not in the document]
Comment (bendikstoasannes) [reply chemie/customprofile/urls.py]: Django even has its own path converter for uuid (Wow!) Will change it to `profil/aktiver/<uuid:code>`

File: chemie/events/templates/events/admin_list.html [hunk 4]
```diff
@@ -237,21 +237,22 @@
...
             if (paymentList[myID][1]){
               changePayment = confirm("Endre betalingsstatus for " +  paymentList[myID][0] + " fra betalt til ikke betalt?")
             }
             if (changePayment){
-              paymentList[myID][1] = !paymentList[myID][1]
+              paymentList[myID][1] = !paymentList[myID][1];
               $.ajax({
                 type: "POST",
-                url: "/events/social/adminlist/paymentstatus/" + myID,
+                url: '{% url 'events:payment_status_social' %}',
```
Comment (anjarfr) [inline chemie/events/templates/events/admin_list.html:248]: nice =D

File: chemie/rentalservice/templates/rentalservice/new_object.html [hunk 5]
```diff
@@ -0,0 +1,21 @@
+{% extends 'chemie/base.html' %}
```
Comment (anjarfr) [inline chemie/rentalservice/templates/rentalservice/new_object.html:1]: Try to avoid doing unrelated changes in the same PR, i guess this should be a commit in the rental service branch instead of here =)

Comment (bendikstoasannes) [reply chemie/rentalservice/templates/rentalservice/new_object.html:1]: This was not the intention. My bad!
